sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -998,6 +998,8 @@ class EthService(
       rewardPercentiles: Array[Double]
   ): EthereumFeeHistoryView = {
     val percentiles = sanitizePercentiles(rewardPercentiles)
+    if (newestBlock == "pending")
```

Comment:
At this point, I think the check on pending can be moved before `sanitizePercentiles` too, so we totally avoid any needless computation.

Reply:
Done